### PR TITLE
update toolbox/toolContainer

### PR DESCRIPTION
**Pull request by thiloSchlemmer**
Adding a container for all tools, which the user can work with

### CODE REVIEW

**Comment by thiloSchlemmer** [comment]
next steps as discussed:

- css by @alsturm 
- refine toolContainer.reducer/action to enable a switch on the tool content

File: package.json [hunk 1]
```diff
@@ -32,15 +32,15 @@
...
     "redux-test-utils": "^1.0.2",
     "style-loader": "^2.0.0",
     "svg-url-loader": "^7.1.1",
     "to-string-loader": "^1.1.6",
     "webpack": "^4.46.0",
     "webpack-bundle-analyzer": "^4.4.0",
     "webpack-cli": "^4.6.0",
-    "webpack-dev-server": "^3.11.2",
-    "portfinder-sync": "0.0.2"
+    "webpack-dev-server": "^3.11.2"
```
Comment (taulinger) [inline package.json:43]: I don't know why this is was modified 🤪
Please have a look at every file before committing it...

**Review by taulinger — COMMENTED** [review]
Very well done!

We could think about giving the ToolContainer content components own folders ...

**Comment by thiloSchlemmer** [comment]
@taulinger  I followed your suggestions and ...

-  moved the content to the appropriate directories 
-  using tag()-Method

**Review by taulinger — APPROVED** [review]
👍

**Pull request merged**